x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/servicenow/servicenow_itsm_params.test.tsx

```diff
@@ -139,33 +139,6 @@ describe('ServiceNowITSMParamsFields renders', () => {
     expect(title.prop('isInvalid')).toBeTruthy();
   });
 
-  test('When subActionParams is undefined, set to default', () => {
```

Comment:
I remove the tests as the core is responsible for providing the default values here: https://github.com/elastic/kibana/pull/117132/files#diff-912e76d0157e4885075c85d7d6c90974539dca2080937eb0371cc133e833701dR34